common/client/node.go

```diff
@@ -175,12 +178,8 @@ func (n *node[CHAIN_ID, HEAD, RPC]) RPC() RPC {
 	return n.rpc
 }
 
-func (n *node[CHAIN_ID, HEAD, RPC]) SubscribersCount() int32 {
-	return n.rpc.SubscribersCount()
-}
-
 func (n *node[CHAIN_ID, HEAD, RPC]) UnsubscribeAllExceptAliveLoop() {
-	n.rpc.UnsubscribeAllExceptAliveLoop()
+	n.rpc.UnsubscribeAllExcept(n.aliveLoopSub)
```

Comment:
We need to also keep finalized block subscription

Reply:
Also access to `aliveLoopSub` is not safe, as `UnsubscribeAllExceptAliveLoop` is called by MultiNode without locking stateMu. 
nit: we should also set aliveLoopSub and finalizedSub to nil after unsubscribing